kafka/scripts/start-kafka.sh

```diff
@@ -17,18 +17,11 @@ fi
 # Set the external host and port
 if [ ! -z "$ADVERTISED_HOST" ]; then
     echo "advertised host: $ADVERTISED_HOST"
-    if grep -q "^advertised.host.name" $KAFKA_HOME/config/server.properties; then
-        sed -r -i "s/#(advertised.host.name)=(.*)/\1=$ADVERTISED_HOST/g" $KAFKA_HOME/config/server.properties
+    if grep -q "^advertised.listeners" $KAFKA_HOME/config/server.properties; then
+        # DIL 2017-10-13 I have no idea why this sed command does not modify the file when run from this script...
```

Comment:
@dllx I will need some more explanation to understand this ...

Reply:
It's the script from upstream, but it wasn't working, so I remove the '#' for which I could not find documentation.
Then I adapted it to the new config key to be used. It still does not replace the value in the config file.
So, I added the comment in deploy that it must remain commented out so the "else echo" branch is executed, which was what worked for the old version (0.10) of our Kafka container.

Everything clear now? :-)